cartography/data/jobs/cleanup/aws_ingest_subnets_cleanup.json

```diff
@@ -0,0 +1,9 @@
+{
+  "statements": [{
+    "query": "MATCH (:AWSAccount{id: {AWS_ID}})-[:RESOURCE]->(:AWSVpc)-[:RESOURCE]->(n:EC2Subnet) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
I think this should be 

```
MATCH (:AWSAccount{id: {AWS_ID}})-[:RESOURCE]->(:AWSVpc)<-[: r:MEMBER_OF_AWS_VPC]-(n:EC2Subnet)
```
instead because it gets merged in this way: https://github.com/lyft/cartography/blob/0c0609362a21ba4f718caff9d3447ea4ca762c5e/cartography/intel/aws/ec2/subnets.py#L39